### PR TITLE
Daemon progress bar redesign

### PR DESCRIPTION
**Pull request by Jaqueeee**
redesigned to fit smaller screens. 
https://github.com/monero-project/monero-core/pull/193

### CODE REVIEW

File: components/DaemonProgress.qml [hunk 1]
```diff
@@ -26,13 +26,13 @@
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import QtQuick 2.0
+import QtQuick 2.6
```
Comment (dEBRUYNE-1) [inline components/DaemonProgress.qml]: @mbg033 can we use anything over 2.0 for QtQuick?

Comment (Jaqueeee) [inline components/DaemonProgress.qml]: 2.6 is available in QT 5.6 and above. Can @mbg033 confirm if QtQuick > 2.0 is an issue?

**Comment by mbg033** [comment]
it depends. if app needs to be built/packaged for e.g. Ubuntu 16.04 - it only provides Qt 5.5.1 in repository.

**Comment by Jaqueeee** [comment]
ok. tnx @mbg033 
I'll find a workaround that doesnt involve > 2.0 then.

**Comment by Jaqueeee** [comment]
@dEBRUYNE-1 qtquick dependency removed.

**Comment by dternyak** [comment]
@Jaqueeee Looks like you have it covered on both counts!

**Pull request merged**